Gemfile

```diff
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
 
-ruby "2.4.3"
+ruby "2.4.4"
 
 gem 'yajl-ruby'
 gem 'librato-metrics', '~> 1.3.0'
-gem 'sinatra'
+gem 'sinatra', '1.4.8'
```

Comment:
@jmervine had to bump sinatra, which I wasn't thrilled out to silence `Fixnum` deprecation errors